e2e-tests/loading-errors/index.test.js

```diff
@@ -1,10 +1,11 @@
-import { baseURL } from "../test-helper";
-
 describe("Validation Errors", () => {
     beforeEach(async () => {
-        await page.goto(`${baseURL}/e2e-tests/loading-errors/index.html`, {
-            waitUntil: "networkidle2",
-        });
+        await page.goto(
+            `http://localhost:${process.env.PORT}/e2e-tests/loading-errors/index.html`,
```

Comment:
Should this use `process.env.BASE_URL`?

Reply:
`process.env.BASE_URL` does make more sense. I'll update it.